transforms/universal/ededup/python/src/ededup_local.py

```diff
@@ -25,7 +25,7 @@
     "output_folder": output_folder,
 }
 
-ededup_params = {"doc_column": "contents", "filter": HashFilter({})}
+ededup_params = {"doc_column": "contents", "doc_id_column": "document_id", "filter": HashFilter({})}
```

Comment:
I think I've asked elsewhere, but can we have a default for "filter" at least in the python case?

Reply:
Its done